packages/fxa-auth-server/package.json

```diff
@@ -182,7 +182,7 @@
     "chai": "^4.3.6",
     "chai-as-promised": "^7.1.1",
     "esbuild": "^0.17.15",
-    "esbuild-register": "^3.2.0",
+    "esbuild-register": "^3.3.3",
```

Comment:
Looks like `esbuild-register` added support for tsconfig paths in 3.3.0.

Reason for not upgrading to latest is that there seemed to be some issues when going to 3.4.0. (Here's [the closed PR](https://github.com/mozilla/fxa/pull/14976))